libindi/drivers/telescope/telescope_script.cpp

```diff
@@ -28,7 +28,6 @@
 #include "telescope_script.h"
 
 #define	POLLMS      1000
-#define STATUS_FILE "/tmp/indi-telesope-gateway.status"
```

Comment:
Wouldn't that be better somewhere in the /var/run? This is essentially a communication point, isn't it?